src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -173,12 +173,16 @@ If a problem occurs, check output in /var/log/proactive/scheduler
 
 === Minimum requirements
 
-The size of a ProActive server is based on the number of jobs and tasks executed and submitted per day. It is also based on the number of nodes connected to the server.
+The sizing of a ProActive server is based on the number of jobs and tasks executed and submitted per day. It is also based on the number of nodes connected to the server.
```

Comment:
I prefer the 'hardware requisites' instead of 'sizing'

Reply:
It doesn't fit "The hardware requirements of a ProActive server is based on the number of jobs". I prefer sizing.